sql/2025/page-weight/use-efficient-cache-lifetimes.sql

```diff
@@ -0,0 +1,18 @@
+-- looks at the distribution of wasted bytes from resources identified by the Lighthouse "Use Efficient Cache Lifetimes" audit
+-- https://developer.chrome.com/docs/performance/insights/cache
+SELECT
+  percentile,
```

Comment:
Should we pivot by root pages? Or want this across both page types?

Reply:
yeah, your right, it should be just root, or both grouped into root and internal.

To be consistent with the rest of the chapter, I've updated the query to group by `is_root_page`